src/js/components/Pagination/stories/Pagination.stories.js

```diff
@@ -0,0 +1,10 @@
+export { Custom } from './Custom';
+export { Grid } from './Grid';
+export { NumberEdgePages } from './NumberEdgePages';
+export { NumberMiddlePages } from './NumberMiddlePages';
+export { Simple } from './Simple';
+export { Size } from './Size';
+
+export default {
+  title: 'Controls/Pagination',
+};
```

Comment:
With the new storybook upgrade, this file should be removed.

Reply:
Removed.